book/content/reproducibility/04/barriers.md

```diff
@@ -40,9 +40,18 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 
 *replace this text with the content of barrier 7*
 
-## Barrier 8
-
-*replace this text with the content of barrier 8*
+## Big data and complex computational infrastructure
+
+Big data is conceptualised in different ways by different researchers. 
+The traditional understanding of big data focuses on complexity, variety of data sources, volume and velocity of the data. 
+It is extremely difficult to control to ensure consistency of data between the experiments. 
+In some cases, when data is sampled from a larger collection of data consistency cannot be guaranteed.
```

Comment:
Tagging @crowcroft because I remember him bringing this up years ago! Jon - do you have any good links to data sources to add to the flight tracking or internet traffic parts of this sentence?

```suggestion
For example, streaming data such as flight tracking or internet traffic can not be stored and must be processed in real time.
```